src/main/resources/db/migration/V1__init_db.sql

```diff
@@ -0,0 +1,154 @@
+CREATE SEQUENCE IF NOT EXISTS category_seq START WITH 1 INCREMENT BY 50;
+
+CREATE SEQUENCE IF NOT EXISTS criteria_seq START WITH 1 INCREMENT BY 50;
+
+CREATE SEQUENCE IF NOT EXISTS interests_seq START WITH 1 INCREMENT BY 50;
+
+CREATE SEQUENCE IF NOT EXISTS places_seq START WITH 1 INCREMENT BY 50;
+
+CREATE SEQUENCE IF NOT EXISTS users_seq START WITH 1 INCREMENT BY 50;
+
+CREATE TABLE category
+(
+    id   BIGINT NOT NULL,
+    name VARCHAR(25),
+    CONSTRAINT pk_category PRIMARY KEY (id)
+);
+
+CREATE TABLE criteria
+(
+    id          BIGINT  NOT NULL,
+    name        VARCHAR(25),
+    deleted     BOOLEAN NOT NULL,
+    interest_id BIGINT,
+    CONSTRAINT pk_criteria PRIMARY KEY (id)
+);
+
+CREATE TABLE interest_category
+(
+    category_id BIGINT NOT NULL,
+    interest_id BIGINT NOT NULL
+);
+
+CREATE TABLE interests
+(
+    id          BIGINT        NOT NULL,
+    name        VARCHAR(25)   NOT NULL,
+    description VARCHAR(1000) NOT NULL,
+    deleted     BOOLEAN       NOT NULL,
+    exclusive   BOOLEAN       NOT NULL,
+    image_name  VARCHAR(255),
+    CONSTRAINT pk_interests PRIMARY KEY (id)
+);
+
+CREATE TABLE likes
+(
+    app_user_id BIGINT NOT NULL,
+    interest_id BIGINT NOT NULL,
+    CONSTRAINT pk_likes PRIMARY KEY (app_user_id, interest_id)
+);
+
+CREATE TABLE place_image_names
+(
+    place_id    BIGINT NOT NULL,
+    image_names VARCHAR(255)
+);
+
+CREATE TABLE places
+(
+    id          BIGINT           NOT NULL,
+    name        VARCHAR(25)      NOT NULL,
+    description VARCHAR(1000),
+    address     VARCHAR(255),
+    latitude    DOUBLE PRECISION NOT NULL,
+    longitude   DOUBLE PRECISION NOT NULL,
+    deleted     BOOLEAN          NOT NULL,
+    user_id     BIGINT,
+    interest_id BIGINT,
+    CONSTRAINT pk_places PRIMARY KEY (id)
+);
+
+CREATE TABLE ratings
+(
+    rating       BIGINT NOT NULL,
+    app_user_id  BIGINT NOT NULL,
```

Comment:
Maybe this can be:

"rating       INT NOT NULL CHECK (rating BETWEEN 1 AND 10),"

probably no need for BIGINT